src/transactions/test/TxEnvelopeTests.cpp

```diff
@@ -47,6 +47,7 @@ TEST_CASE("txset - correct apply order", "[tx][envelope]")
 {
     Config cfg = getTestConfig();
     cfg.NODE_SEED = SecretKey::fromSeed(sha256("NODE_SEED"));
+    cfg.TESTING_UPGRADE_LEDGER_PROTOCOL_VERSION = 18;
```

Comment:
Why protocol version 18?

Reply:
I removed this line (I think it failed in my protocol 19 branch, but that can be fixed later).